provisioning/image_customization.go

```diff
@@ -157,6 +158,10 @@ func createImageCustomizationContainer(images *Images, info *ProvisioningInfo, i
 				Name:  ipOptions,
 				Value: info.NetworkStack.IpOption(),
 			},
+			corev1.EnvVar{
+				Name:  additionalNTPServers,
+				Value: strings.Join(info.ProvConfig.Spec.AdditionalNTPServers[:], ","),
```

Comment:
I'm surprised to see `[:]`.

Reply:
Slip of the tongue (where tongue = ctrl+v). Fixed.